riff-raff/app/controllers/DeployController.scala

```diff
@@ -348,14 +348,14 @@ class DeployController(
     }
 
     val params = DeployParameterForm(
-      record.buildName,
-      record.buildId,
-      "",
```

Comment:
The stage and branch fields were being passed in the wrong order in both of these controllers. This led to [strange behaviour/errors](https://riffraff.gutools.co.uk/deployment/view/7513d323-9fda-4189-b8c4-ffc4b307f677). This PR should fix the problem and also uses named parameters to make it clearer what's going on in these cases. 

Unfortunately it's easy to make this type of mistake when we are just passing around lots of unnamed `String`s!